### PR TITLE
update transaction struct log

### PR DESCRIPTION
**Pull request by millken**

### CODE REVIEW

**Comment by dustinxie** [comment]
can we not change every pb.go file each time?

when I run `make gogen`, i only saw affected files being changed, not all files

**Comment by millken** [comment]
> can we not change every pb.go file each time?
> 
> when I run `make gogen`, i only saw affected files being changed, not all files

done, update only changed pb.go

File: proto/types/transaction_log.proto [hunk 1]
```diff
@@ -45,8 +45,8 @@ enum TransactionLogType {
 message TransactionStructLog {
```
Comment (Liuhaai) [inline proto/types/transaction_log.proto:45]: Here is struct from geth
```
type StructLogRes struct {
	Pc      uint64             `json:"pc"`
	Op      string             `json:"op"`
	Gas     uint64             `json:"gas"`
	GasCost uint64             `json:"gasCost"`
	Depth   int                `json:"depth"`
	Error   string             `json:"error,omitempty"`
	Stack   *[]string          `json:"stack,omitempty"`
	Memory  *[]string          `json:"memory,omitempty"`
	Storage *map[string]string `json:"storage,omitempty"`
}
```
Could you check whether our implementation is compatible with theirs?

**Comment by dustinxie** [comment]
> > can we not change every pb.go file each time?
> > when I run `make gogen`, i only saw affected files being changed, not all files
> 
> done, update only changed pb.go

after discussing, let's move to newer protoc version, and keep all changes

File: proto/types/transaction_log.proto (hunk 1 above)
Comment (millken) [reply proto/types/transaction_log.proto:45]: Currently except `Storage` field, everything else is the same.

**Review by dustinxie — APPROVED** [review]

**Pull request merged**